scrunch/datasets.py

```diff
@@ -2393,27 +2404,37 @@ def fork(self, description=None, name=None, is_published=False,
             is_published=is_published,
             **kwargs
         )
-       
+
         # Handling project vs owner conflict
         owner = kwargs.get("owner")
 
         if project and owner:
             raise ValueError(
                 "Cannot pass both 'project' & 'owner' parameters together. "
                 "Please try again by passing only 'project' parameter."
-                )
+        )
         elif owner:
             project = owner
-        
+
         # Setting project value based on preserve_owner.
-        if preserve_owner and project:
-            raise ValueError("Cannot pass 'project' or 'owner' when preserve_owner=True")
-        elif preserve_owner:
-            body["owner"] = self.resource.body.owner
-        elif project:
-            body["owner"] = (
-                project if project.startswith("http") else get_project(project).url
-            )
+        if preserve_owner:
+            if project:
+                raise ValueError(
+                    "Cannot pass 'project' or 'owner' when preserve_owner=True."
+                )
+            else:
+                # Create fork in source dataset path.
+                body["owner"] = self.resource.body.owner
```

Comment:
you can replace `body["owner"]` with `body["project"]` to future proof this

Reply:
Done